libraries/Microsoft.Bot.StreamingExtensions/OperationException.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Bot.StreamingExtensions
+{
+    internal class OperationException : Exception
+    {
```

Comment:
The custom exception type doesn't make sense to me. The general .Net pattern is to not create new exception types anymore. If we *really* need one here, please explain what it's doing.